statsmodels/discrete/conditional_models.py

```diff
@@ -13,18 +13,27 @@
 import itertools
 
 
-
 class conditionalModel(base.LikelihoodModel):
```

Comment:
conditionalModel -> ConditionalModel?

Reply:
This was intended to be a private class, I guess the name should be `_ConditionalModel` then.